addon/components/validated-input/template.hbs

```diff
@@ -39,9 +39,8 @@
   <div class="{{config.css.checkbox}}">
     <label>
       {{one-way-checkbox
-          checked
           name       = name
-          value      = value
+          checked    = (get model name)
```

Comment:
I still don't agree that this is the right way. For a checkbox there is a separation between `value` and `checked` property. The value of the `checked` attribute is either false (or the attribute is missing) or is anything else. I believe jQuery's `.val()` returns the value of the `value` attribute of the checkbox, not the value of `checked`.
So you can't lump them together. There is a clear use case where you want to know the values of all the options checked in a group, which form an array. Not having values set on checkboxes breaks this case.

More so, even if you did have a checkbox group, this component would not be usable inside the checkbox group component.  I think a much better solution is to have:
```hbs
{{one-way-checkbox
value = (get model name)
checked = checked
...
}}
```
so that the checkboxGroup would control the checked property.